superset-frontend/src/explore/components/ControlPanelsContainer.jsx

```diff
@@ -139,6 +139,10 @@ class ControlPanelsContainer extends React.Component {
           controls[s].validationErrors.length > 0,
       ),
     );
+    if (exploreState.dataset_health_check_message) {
```

Comment:
i can populate from here, or from `superset-ui-chart-controls`:
https://github.com/apache-superset/superset-ui/blob/1f8a700b953efbed81d2c9970b514d7fe031ad2e/packages/superset-ui-chart-controls/src/shared-controls/index.tsx#L166

@ktmud

Reply:
This component `ControlPanelsContainer` is generic and should stay generic. It should not contain logics related to any specific control field.